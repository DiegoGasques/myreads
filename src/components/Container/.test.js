import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";

import Container from "./index";

it("Should match the component snapshot", () => {
  const wrapper = shallow(<Container />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
