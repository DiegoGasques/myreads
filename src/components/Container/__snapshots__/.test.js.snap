// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match the component snapshot 1`] = `
<div
  className="container"
/>
`;
